external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -13,20 +12,34 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageToInclude Include="xunit.abstractions"/>
+    <PackageToInclude Include="xunit.assert"/>
+    <PackageToInclude Include="xunit.extensibility.core"/>
+    <PackageToInclude Include="xunit.extensibility.execution"/>
+    <PackageToInclude Include="xunit.runner.utility"/>
+    <PackageToInclude Include="Microsoft.xunit.netcore.extensions"/>
+    <PackageToInclude Include="Microsoft.DotNet.xunit.performance.run.core"/>
+    <PackageToInclude Include="Newtonsoft.Json"/>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetGroup)' != 'netfx'">
+    <PackageToInclude Include="xunit.console.netcore"/>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
+    <PackageToInclude Include="xunit.runner.console"/>
+  </ItemGroup>
+
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-  <Target Name="FilterNugetPackage" AfterTargets="ResolveNuGetPackages">
-    <!-- Filter down to just xunit assets -->
+
+  <Target Name="AddXunitConsoleRunner" BeforeTargets="ResolveReferences" Condition="'$(TargetGroup)' == 'netfx'">
     <ItemGroup>
-      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.abstractions' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.assert' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.extensibility.core' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.extensibility.execution' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.console.netcore' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='xunit.runner.utility' And 
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='Microsoft.xunit.netcore.extensions' And
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='Microsoft.DotNet.xunit.performance.run.core' And
-                                                                              '%(ReferenceCopyLocalPaths.NuGetPackageId)'!='Newtonsoft.Json'
-                                                                              " />
+      <ReferenceCopyLocalPaths Include="$(PackagesDir)xunit.runner.console\2.2.0-beta2-build3300\tools\*.*">
```

Comment:
Put this version in a property and share in in this project. Then do an exist check for this directory and error if it doesn't exist.